pkg/node/volumemgr.go

```diff
@@ -1263,6 +1263,7 @@ func (m *VolumeManager) isRootMountpoint(devs []lsblk.BlockDevice) bool {
 
 // addVolumeStatusAnnotation add annotation with volume status to drive
 func (m *VolumeManager) addVolumeStatusAnnotation(drive *drivecrd.Drive, volumeName, status string) {
+	m.log.Infof("Adding volume status annotation %s: %s", volumeName, status)
```

Comment:
Will the drive be visible somehow in this log? if not please log the drive name or id as well.